source/_integrations/shelly.markdown

```diff
@@ -330,6 +330,10 @@ The integration supports the following virtual components:
 - `text` in `field` mode, for which a `text` platform entity is created
 - `text` in `label` mode, for which a `sensor` platform entity is created
 
+## Scripts
```

Comment:
```suggestion
## Scripts (generation 2 and 3)
```

Reply:
I just mutuate what we did for Virtual Components that are lacking the gen